src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/IPCServicesTest.java

```diff
@@ -136,6 +144,14 @@ void beforeEach(ExtensionContext context) {
         ignoreExceptionOfType(context, InterruptedException.class);
     }
 
+    @DisabledOnOs(OS.WINDOWS)
```

Comment:
How did we verify this works on Windows?

Reply:
Setting a custom path does _not_ work on Windows. We still use a named pipe as before. The tests run on Windows showing that it is unaffected.

Reply:
Are there tests that also exercise the default behavior without customized path?